src/olympia/users/models.py

```diff
@@ -479,27 +479,57 @@ def ban_and_disable_related_content_bulk(cls, users, move_files=False):
         from olympia.ratings.models import Rating
         from olympia.users.tasks import delete_photo
```

Comment:
just checking that changing `delete_photo` to soft-delete is planned in a follow-up.

Reply:
Yeah, that's part of https://github.com/mozilla/addons-server/issues/21445 - I didn't want to block the relationship soft-deletion (or even unbanning itself) on the image handling